src/Dog.scala

```diff
@@ -0,0 +1,15 @@
+/**
+  * Created by halleyfroeb on 10/17/16.
+  */
+class Dog {
+  var name = ""
+  var breed = ""
+  var isAlive = true
+
+  def printFields(): Unit={
+    var printAlive = ""
+    if(isAlive) printAlive = "is Alive" else printAlive = "is deceased"
+    println(s"$name is a $breed and $printAlive\n")
+  }
```

Comment:
Instead of printFields() how about overriding the toString(). That would be more typical.  Same logic but you'd just be returning a String to let the caller decide how to handle it, whether to print or whatever.